src/test/resources/org/jenkinsci/plugins/github/config/configuration-as-code.yml

```diff
@@ -0,0 +1,17 @@
+unclassified:
+
+  githubpluginconfig:
```

Comment:
maybe another more minimal test which shows what you need if you set manage hooks false and no hook url?